### DISCUSSION TITLE
Snapper grub

### DISCUSSION THREAD

**Discussion by robertocannito** [SpiralLinux/SpiralLinux-project#217] [General]
Hi with Sid after a fresh installation, on boot grub menu "snapper menu" is not more available.

how fix it?

**Comment by geckolinux** [comment]
Thanks for reporting. It looks like it's due to an upstream change in `btrfs-progs` :
https://github.com/Antynea/grub-btrfs/issues/254

Hopefully a fixed version will appear in Sid at some point. Meanwhile you can downgrade `btrfs-progs` to this version:
https://snapshot.debian.org/archive/debian/20221201T090253Z/pool/main/b/btrfs-progs/btrfs-progs_6.0.2-1_amd64.deb

**Reply by dirtydog01** [reply]
Thanks

**Comment by robertocannito** [comment]
great Sam it works after a grub-update

thanks a lot and happy new year, all the best
